bibdk.make

```diff
@@ -17,10 +17,15 @@ libraries[ckeditor][download][url] = http://download.cksource.com/CKEditor/CKEdi
 
 libraries[slick][destination] = libraries
 libraries[slick][directory_name] = slick
-libraries[slick][download][type] = git
+; libraries[slick][download][type] = git
 libraries[slick][download][url] = https://github.com/kenwheeler/slick/
 libraries[slick][download][tag] = '1.8.0'
 
+libraries[audit][destination] = libraries
+libraries[audit][directory_name] = audit
+libraries[audit][download][type] = git
+libraries[audit][download][url] = gitlab@gitlab.dbc.dk:pu/audit/audit-trail-php-library.git
+libraries[audit][download][branch] = 'codereview'
```

Comment:
Går ud fra at dette er det samme som for de andre produkter, og vil skifte til master, når den er klar?

Reply:
Jeps, det er simpelthen for at jeg kunne teste. Retter det så snart det andet MR er godkendt.